web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -412,31 +408,4 @@ public static String formatDate(Date date, Locale locale, FORMAT_TYPE type) {
 		return date == null ? "" : dateFormat.format(date);
 	}
 	
-	/**
-	 * @see org.openmrs.api.GlobalPropertyListener#supportsPropertyName(java.lang.String)
-	 */
-	@Override
-	public boolean supportsPropertyName(String propertyName) {
-		return OpenmrsConstants.GP_SEARCH_DATE_DISPLAY_FORMAT.equals(propertyName);
```

Comment:
This method has nothing to do with defaultDateCache. So removing the implemented interface may have unintended consequences.